lib/main.go

```diff
@@ -214,7 +215,29 @@ func (obj *Main) Run() error {
 		return fmt.Errorf("hostname cannot be empty")
 	}
 
+	// Get current user running mgmt
+	user, err := user.Current()
+	if err != nil {
+		return errwrap.Wrapf(err, "can't get current user")
+	}
+
+	// Use systemd SateDirectory if set. If not, use
+	// XDG_CACHE_DIR unless user is root, then use
+	// /var/lib/mgmt
```

Comment:
dir's (even in comments) have a trailing slash please, and full sentences end with a period.

(Make sure comments are wrapped to 80 chars please.)